__mocks__/@grafana/ui.tsx

```diff
@@ -40,11 +40,20 @@ const MultiSelect = forwardRef((props, ref) => <Select {...props} ref={ref} mult
 
 MultiSelect.displayName = 'MultiSelect';
 
+// Using this specifically for Probe filter at the moment
```

Comment:
So now that I think about it, if you're mocking the datasource in your tests (which I think you are), then we _could_ theoretically just call the `loadOptions` prop and pass that as options to the mocked select. Probably further down the rabbit hole than we need to go right now, just thinking out loud